skaffold.yaml

```diff
@@ -1,141 +1,164 @@
+  - name: azure-setup
+    patches:
+      - op: add
+        path: /manifests/kustomize/paths
+        value: [ hack/e2e-test/infrastructure/overlays/azure/setup ]
+  # Profile to delete the Azure storage container from an e2e test.
+  - name: azure-cleanup
+    patches:
+      - op: add
+        path: /manifests/kustomize/paths
+        value: [ hack/e2e-test/infrastructure/overlays/azure/cleanup ]
+  # Profile to create the required GCP backup bucket for an e2e test.
+  - name: gcp-setup
+    patches:
+      - op: add
+        path: /manifests/kustomize/paths
+        value: [ hack/e2e-test/infrastructure/overlays/gcp/setup ]
+      - op: add
+        path: /manifests/kustomize/hooks/before/-
+        value: {
+          "host": {
+            "command": [ sh, -c, '
       echo "Copying GCP service account json" &&
       touch "hack/e2e-test/infrastructure/overlays/gcp/common/assets/serviceaccount.json" &&
-      cp "$GCP_SERVICEACCOUNT_JSON_PATH" "hack/e2e-test/infrastructure/overlays/gcp/common/assets/serviceaccount.json"'],
-        "os": [darwin, linux]
-      }
-    }
-# Profile to delete the GCP backup bucket from an e2e test.
-- name: gcp-cleanup
-  patches:
-  - op: add
-    path: /deploy/kustomize/paths
-    value: [hack/e2e-test/infrastructure/overlays/gcp/cleanup]
-  - op: add
-    path: /deploy/kustomize/hooks/before/-
-    value: {
-      "host": {
-        "command": [sh, -c, '
+      cp "$GCP_SERVICEACCOUNT_JSON_PATH" "hack/e2e-test/infrastructure/overlays/gcp/common/assets/serviceaccount.json"' ],
+            "os": [ darwin, linux ]
+          }
+        }
+  # Profile to delete the GCP backup bucket from an e2e test.
+  - name: gcp-cleanup
+    patches:
+      - op: add
+        path: /manifests/kustomize/paths
+        value: [ hack/e2e-test/infrastructure/overlays/gcp/cleanup ]
+      - op: add
+        path: /manifests/kustomize/hooks/before/-
+        value: {
+          "host": {
+            "command": [ sh, -c, '
       echo "Copying GCP service account json" &&
       touch "hack/e2e-test/infrastructure/overlays/gcp/common/assets/serviceaccount.json" &&
-      cp "$GCP_SERVICEACCOUNT_JSON_PATH" "hack/e2e-test/infrastructure/overlays/gcp/common/assets/serviceaccount.json"'],
-        "os": [darwin, linux]
-      }
-    }
-# Profile to create the required Local storage container for an e2e test.
-- name: local-setup
-  patches:
-    - op: add
-      path: /deploy/kustomize/paths
-      value: [hack/e2e-test/infrastructure/overlays/local/setup]
-# Profile to delete the Local storage container from an e2e test.
-- name: local-cleanup
-  patches:
-    - op: add
-      path: /deploy/kustomize/paths
-      value: [hack/e2e-test/infrastructure/overlays/local/cleanup]
+      cp "$GCP_SERVICEACCOUNT_JSON_PATH" "hack/e2e-test/infrastructure/overlays/gcp/common/assets/serviceaccount.json"' ],
+            "os": [ darwin, linux ]
+          }
+        }
+  # Profile to create the required Local storage container for an e2e test.
```

Comment:
Could YAML syntax be used instead of JSON suggested below? The LSP for YAML complains about ill formated YAML.
```suggestion
      - op: add
        path: /manifests/kustomize/hooks/before/-
        value:
          host:
            command:
              - sh
              - -c
              - |
                echo "Copying GCP service account json" &&
                touch "hack/e2e-test/infrastructure/overlays/gcp/common/assets/serviceaccount.json" &&
                cp "$GCP_SERVICEACCOUNT_JSON_PATH" "hack/e2e-test/infrastructure/overlays/gcp/common/assets/serviceaccount.json"
            os:
              - darwin
              - linux
  # Profile to delete the GCP backup bucket from an e2e test.
  - name: gcp-cleanup
    patches:
      - op: add
        path: /manifests/kustomize/paths
        value: [ hack/e2e-test/infrastructure/overlays/gcp/cleanup ]
      - op: add
        path: /manifests/kustomize/hooks/before/-
        value: 
          host: 
            command:
              - sh
              - -c
              - |
                echo "Copying GCP service account json" &&
                touch "hack/e2e-test/infrastructure/overlays/gcp/common/assets/serviceaccount.json" &&
                cp "$GCP_SERVICEACCOUNT_JSON_PATH" "hack/e2e-test/infrastructure/overlays/gcp/common/assets/serviceaccount.json" 
            os:
              - darwin
              - linux
```

Reply:
We will perhaps remove provider specific e2e tests from druid. This would mean that the above would also be eventually removed. I would recommend that we do not change this now as it will be removed in the near future.